indico/modules/events/registration/forms.py

```diff
@@ -423,3 +424,33 @@ class RejectRegistrantsForm(IndicoForm):
 
     def is_submitted(self):
         return super().is_submitted() and 'submitted' in request.form
+
+
+class RegistrationTagForm(IndicoForm):
+    """Form to create a new registration tag."""
+
+    title = StringField(_('Title'), [DataRequired()])
+    color = StringField(_('Color'), [DataRequired()], widget=ColorInput())
+
+    def __init__(self, *args, **kwargs):
+        self.event = kwargs.pop('event', None)
+        super().__init__(*args, **kwargs)
+
+    def validate_title(self, field):
```

Comment:
I think you're missing the logic for editing here - changing a tag's color will most likely fail the validation in here because the current check will think the title conflicts with itself.